packages/ubus-lime-metrics/files/usr/libexec/rpcd/lime-metrics

```diff
@@ -105,6 +126,21 @@ local function get_gateway(msg)
    printJson(_get_gateway())
 end
 
+local function get_last_internet_path(msg)
+    local path = list_from_file("/etc/last_internet_path")
```

Comment:
This information looks like quite volatile, and to store it in persistent flash memory cold cause flash breakage to many write cycles, is it necessary to store this information persistently?

Reply:
It is not really volatile information, it is information that must persist when the router restarts. Suppose that the user loses the Internet connection. The first thing they do is to restart the router, or if an electrical outage occurs in part of the network, etc. In order to be able to diagnose what happens with your usual exit to interent you need to be able to remember where you used to go.